Adding-a-new-device.md

```diff
@@ -0,0 +1,47 @@
+## Adding a new device
+
+Use the cli (described in the zwave adding a new device document).
```

Comment:
zwave? is it the same process?

Reply:
Yeah - the cli program could basically be used for any adapter. The device/info commands show the device dictionary that the adapter exposes and the thing/info commands show the thing descriptions. Command like discover use the debugCmd mechanism which is adapter agnostic, but the actual discover command is only recognized by the zigee adapter (for example).